cram_pr2/cram_pr2_shopping_demo/resource/basket_urdf.urdf

```diff
@@ -0,0 +1,149 @@
+            </geometry>
+        </collision>
+        <visual>
+            <geometry>
+                <box size="0.3 0.005 0.18"/>
+            </geometry>
+        </visual>
+    </link>
+
+    <joint name="bottom_to_top_side_joint" type="fixed">
+        <origin xyz="0 -0.25 0.09"/>
+        <parent link="bottom_link"/>
+        <child link="top_side_link"/>
+    </joint>
+
+    <link name="right_side_link">
+        <collision>
+            <geometry>
+                <box size="0.005 0.50 0.18"/>
+            </geometry>
+        </collision>
+        <visual>
+            <geometry>
+                <box size="0.005 0.50 0.18"/>
+            </geometry>
+        </visual>
+    </link>
+
+    <joint name="bottom_to_right_side_joint" type="fixed">
+        <origin xyz="-0.15 0 0.09"/>
+        <parent link="bottom_link"/>
+        <child link="right_side_link"/>
+    </joint>
+
+    <link name="bottom_side_link">
+        <collision>
+            <geometry>
+                <box size="0.3 0.005 0.18"/>
+            </geometry>
+        </collision>
+        <visual>
+            <geometry>
+                <box size="0.3 0.005 0.18"/>
+            </geometry>
+        </visual>
+    </link>
+
+    <joint name="bottom_to_bottom_side_joint" type="fixed">
+        <origin xyz="0 0.25 0.09"/>
+        <parent link="bottom_link"/>
+        <child link="bottom_side_link"/>
+    </joint>
+
+    <link name="handle_left_link">
+        <collision>
+            <geometry>
+                <box size="0.005 0.005 0.09"/>
+            </geometry>
+        </collision>
+        <visual>
+            <geometry>
+                <box size="0.005 0.005 0.09"/>
+            </geometry>
+        </visual>
+    </link>
+
+    <joint name="handle_left_to_bottom_joint" type="fixed">
+        <origin xyz="0.15 0 0.225"/>
+        <parent link="bottom_link"/>
+        <child link="handle_left_link"/>
+    </joint>
+
+    <link name="handle_right_link">
+        <collision>
+            <geometry>
+                <box size="0.005 0.005 0.09"/>
+            </geometry>
+        </collision>
+        <visual>
+            <geometry>
+                <box size="0.005 0.005 0.09"/>
+            </geometry>
+        </visual>
+    </link>
+
+    <joint name="handle_right_to_bottom_joint" type="fixed">
+        <origin xyz="-0.15 0 0.225"/>
+        <parent link="bottom_link"/>
+        <child link="handle_right_link"/>
+    </joint>
+
+    <link name="handle_middle_link">
+        <collision>
+            <geometry>
+                <box size="0.3 0.005 0.005"/>
+            </geometry>
+        </collision>
+        <visual>
+            <geometry>
+                <box size="0.3 0.005 0.005"/>
+            </geometry>
+        </visual>
+    </link>
+
+    <joint name="handle_middle_to_bottom_joint" type="fixed">
+        <origin xyz="0 0 0.27"/>
+        <parent link="bottom_link"/>
+        <child link="handle_middle_link"/>
+    </joint>
+
+</robot>
```

Comment:
Why were the stl files removed? Like `denkmit.stl`, `dove.stl` and `heitmann.stl`?

Reply:
They weren't removed, I decimated the shaped and because of this the files changed